pom.xml

```diff
@@ -105,6 +105,13 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-stream</artifactId>
 		</dependency>
+		<!-- Lattice core dependency that activates cloud,lattice profiles when running on Lattice -->
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-lattice-core</artifactId>
+			<version>${spring-cloud-lattice.version}</version>
```

Comment:
We don't have `spring-cloud-starters` specified in `dependencyManagement` for `s-c-s-modules` since we don't need that for any sub projects but only for the parent POM. Hence the explicit version here.